src/components/NewPlayerForm.js

```diff
@@ -1,16 +1,37 @@
 import React, { useState } from 'react'
+import injectSheet from 'react-jss'
 import axios from 'axios'
 
 import { API_URL, CREATE } from 'constants/api'
 
 import Form from 'components/Form'
 
+import { robotoBold } from 'constants/styles/fonts'
+import { yellow } from 'constants/styles/colors'
+
+const styles ={
+	formTitle: {
+		textTransform: 'uppercase',
+		...robotoBold,
+	},
+	nameInput: {
+		border: '2px solid black',
+		fontSize: 12,
+		padding: 8,
+		backgroundColor: yellow,
+		'&:focus': {
+			outline: 'none',
+			backgroundColor: '#fff',
```

Comment:
Make and import a `white` constant for this, even though we all know what `#fff` is